benchmark/SerializerBenchmark/Serializers/OdinSerializer.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) All contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+extern alias odin;
```

Comment:
Drop this when you remove the `Aliases` metadata from the reference.

Reply:
At the time the suite didn't recognize Odin without the alias. But now it works fine for some reason, so sure.